app/(aux)/circleSettings.js

```diff
@@ -385,11 +397,11 @@ export default function Page() {
 								iconSize={60}
 								icon='musical-notes'
 								iconColor='white'
-								href='/mainViewLayout'
+								href='/'
 							/>
 
 							<StyledText className='top-[20%] text-3xl text-offwhite'>
-								Circle Name
+								Circle Namec
```

Comment:
typo here but also this can be updated to display the current circle name

Reply:
[Done]

Reply:
[done]